scripts/smb_multijoin.py

```diff
@@ -57,7 +57,7 @@ def mkCogroupFnRetVal(n, aWrapper=None, otherWrapper=None):
 
 def mkTransformFnRetVal(n, aWrapper=None, otherWrapper=None):
     vals = fnRetValHelper(n, aWrapper, otherWrapper)
-    return 'SortedBucketScioContext#SortMergeTransformReadBuilder[KEY, (%s)]' % ', '.join(vals)
+    return 'SortedBucketScioContext#SortMergeTransformReadBuilder[KEY, KEY, Void, (%s)]' % ', '.join(vals)
```

Comment:
was this call outdated?

Reply:
this is what was changed manually when adding the secondary SMB sort key I believe